sonar-scanner-engine/src/main/java/org/sonar/batch/bootstrapper/IssueListener.java

```diff
@@ -19,6 +19,10 @@
  */
 package org.sonar.batch.bootstrapper;
 
+/**
+ * @deprecated since 6.2 was used by initial version of SonarLint. No more used.
+ */
+@Deprecated
 @FunctionalInterface
 public interface IssueListener {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)